components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/internal/EntitlementServiceComponent.java

```diff
@@ -307,6 +317,36 @@ protected void activate(ComponentContext ctxt) {
         }
     }
 
+    /**
+     * Adds policy files with unique policyIDs to the registry.
+     *
+     * @param policyIdList
+     * @param fileList
+     * @return
+     * @throws IOException
+     */
+    private boolean addPolicyFiles(List<String> policyIdList, File[] fileList) throws IOException {
+
+        boolean customPolicies = false;
+        for (File policyFile : fileList) {
+            if (policyFile.isFile()) {
+                PolicyDTO policyDTO = new PolicyDTO();
+                policyDTO.setPolicy(FileUtils.readFileToString(policyFile));
+                if (!policyIdList.contains(policyDTO.getPolicyId())) {
+                    try {
+                        EntitlementUtil.addFilesystemPolicy(policyDTO, registryService
+                                .getGovernanceSystemRegistry(), true);
+                    } catch (Exception e) {
+                        // log and ignore
```

Comment:
Format the comment by starting with a capital letter and ending with a '.'. And add a meaningful sentence here.

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3110